<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskDialog</class>
 <widget class="QDialog" name="TaskDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>361</width>
     <height>337</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="titleEdit">
      <property name="placeholderText">
       <string>Add a task</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLineEdit" name="newTagLineEdit"/>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="addTagButton">
      <property name="text">
       <string>Add Tag</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="tagLabel">
      <property name="text">
       <string>Tags</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QDateEdit" name="dueDateEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="statusComboBox">
      <item>
       <property name="text">
        <string>To Do</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>In Progress</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Done</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="newTagLabel">
      <property name="text">
       <string>Add a new tag</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QListWidget" name="tagsListWidget"/>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="removeTagButton">
      <property name="text">
       <string>Remove Tag</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TaskDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TaskDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
